milestone-3/frontend/src/components/recipePage.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import './recipePage.css'
+import IngredientsList from './ingredientsList';
+import InstructionsList from './instructionsList';
+import InputForm from './inputForm';
+
+export default function RecipePage({recipeName, recipeDesc, imageSrc, ingredients, servingSize, instructions}) {
+
+  const [ingredientsList, addIngredient] = React.useState(ingredients);
```

Comment:
A naming convention for state variables is [state, setState]. So in this case, [ingredientsList, setIngredientsList] is a slightly better name. We'll go over the UseState hook tomorrow in more depth!